components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -49,7 +49,7 @@
      * @param user       Username of the comment author
      * @throws APIManagementException if failed to add comment for API
      */
-    String addComment(Identifier identifier, Comment comment, String user) throws APIManagementException;
+    String addComment(String identifier, Comment comment, String user) throws APIManagementException;
```

Comment:
parameter name should be uuid

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/10614/commits/b561990adfc350c2aec76fc8e72c88c6ad896f15